crates/astria-composer/src/executor/bundle_factory/mod.rs

```diff
@@ -81,6 +83,9 @@ impl SizedBundle {
         let seq_action_size = estimate_size_of_sequence_action(&seq_action);
 
         if seq_action_size > self.max_size {
+            metrics::gauge!(crate::metrics_init::TRANSACTIONS_DROPPED_TOO_LARGE,
```

Comment:
IMO this metric is too deep down in the stack. I would emit it at the level that is handling this error

Reply:
Have moved it to a higher level.